MatrixSDK/Data/RoomList/CoreData/MXCoreDataRoomListDataFetcher.swift

```diff
@@ -114,10 +113,8 @@ internal class MXCoreDataRoomListDataFetcher: NSObject, MXRoomListDataFetcher {
         return result
     }
     
-    internal init(session: MXSession?,
-                  fetchOptions: MXRoomListDataFetchOptions,
+    internal init(fetchOptions: MXRoomListDataFetchOptions,
```

Comment:
Unrelated change, the `session` is never used so can be removed from init parameters